pkg/commands/config.go

```diff
@@ -49,6 +49,7 @@ func newConfigCommand(log logutils.Log, info BuildInfo) *configCommand {
 		Args:              cobra.NoArgs,
 		ValidArgsFunction: cobra.NoFileCompletions,
 		RunE:              c.executeVerify,
+		SilenceUsage:      true,
```

Comment:
Not sure if it's worth adding `SilenceErrors`:

```suggestion
		SilenceUsage:      true,
		SilenceErrors:       true,
```

At the moment this repeats an error message:

```
jsonschema: "linters-settings.govet" does not validate with "/properties/linters-settings/properties/govet/additionalProperties": additionalProperties 'check-shadowing' not allowed
Error: the configuration contains invalid elements
Failed executing command with error: the configuration contains invalid elements
```

("the configuration contains invalid elements" shown twice) but I also have no idea if there are deeper implications to adding this option :upside_down_face: